documentation/12-configuration/README.md

```diff
@@ -0,0 +1,266 @@
+Title: Configuration
+----
+Text:
+
+ContainerPilot expects a configuration file with details about what services it should register, how to check their health, and what to do at startup and shutdown, among others.
+
+### Specifying the configuration file
+
+There are a few ways to specify the ContainerPilot configuration location.
+
+1. An environment variable set in the `docker run...` string, in a Docker Compose manifest, in the Dockerfile, or elsewhere
+2. As an argument to ContainerPilot, perhaps in the the `docker run...` string, or in in the `CMD` or `ENTRYPOINT` of the Dockerfile
+3. As a JSON string passed to ContainerPilot in any of the locations named above
+
+#### Examples: specifying the configuration file path
+
+```bash
```

Comment:
The syntax highlighting for this is displaying black-on-black in the preview container.